nomenclature/processor/region.py

```diff
@@ -394,6 +407,20 @@ def apply(self, df: IamDataFrame, dsd: DataStructureDefinition) -> IamDataFrame:
                             if var not in vars_default_args
                         }
                         for cr in self.mappings[model].common_regions:
+                            # If the common region is only comprised of a single model
+                            # native region, just rename
+                            if (
+                                cr.is_single_constituent_region
+                                and not model_df.filter(
+                                    region=cr.constituent_regions[0]
+                                ).empty
+                            ):
+                                _processed_dfs.append(
+                                    model_df.filter(
+                                        region=cr.constituent_regions[0]
+                                    ).rename(region=cr.rename_dict)
+                                )
+                                continue
```

Comment:
Please add docstring/comment with "else" for clarity

Reply:
added